seed/static/seed/partials/inventory_detail_analyses_modal.html

```diff
@@ -22,7 +22,7 @@ <h4 class="modal-title" id="manageLabelsModalLabel">Run Analysis</h4>
 
                 <div class="form-group" ng-if="new_analysis.service == 'BSyncr'">
                     <label class="control-label sectionLabel">BSyncr Model Selection</label>
-                    <select class="form-control" ng-model="new_analysis.configuration" ng-dropdown required>
+                    <select class="form-control" ng-model="new_analysis.configuration.model_type" ng-dropdown required>
```

Comment:
previously we were overwriting the entire configuration object which isn't what we want